gtm/src/template-parameters.json

```diff
@@ -0,0 +1,404 @@
+        "type": "LABEL"
+      },
+      {
+        "enablingConditions": [
+          {
+            "paramName": "eventType",
+            "type": "EQUALS",
+            "paramValue": "viewedObjectIDs"
+          }
+        ],
+        "displayName": "Sends a view event to capture the items.",
+        "name": "labelViewedObjectIDs",
+        "type": "LABEL"
+      },
+      {
+        "enablingConditions": [
+          {
+            "paramName": "eventType",
+            "type": "EQUALS",
+            "paramValue": "viewedFilters"
+          }
+        ],
+        "displayName": "Sends a view event to capture the filters.",
+        "name": "labelViewedFilters",
+        "type": "LABEL"
+      }
+    ]
+  },
+  {
+    "enablingConditions": [
+      {
+        "paramName": "eventType",
+        "type": "EQUALS",
+        "paramValue": "init"
+      }
+    ],
+    "displayName": "",
+    "name": "Init Setting",
+    "groupStyle": "NO_ZIPPY",
+    "type": "GROUP",
+    "subParams": [
+      {
+        "displayName": "App ID",
+        "help": "The identifier of your Algolia application.",
+        "simpleValueType": true,
+        "name": "appId",
+        "type": "TEXT"
+      },
+      {
+        "displayName": "API Key",
+        "help": "The search API key of your Algolia application.",
+        "simpleValueType": true,
+        "name": "apiKey",
+        "type": "TEXT"
+      }
+    ]
+  },
+  {
+    "enablingConditions": [
+      {
+        "paramName": "eventType",
+        "type": "EQUALS",
+        "paramValue": "init"
+      }
+    ],
+    "displayName": "Advanced Algolia Settings",
+    "name": "Advanced Algolia Settings",
+    "groupStyle": "ZIPPY_CLOSED",
+    "type": "GROUP",
+    "subParams": [
+      {
+        "displayName": "Opt Out User",
+        "name": "userHasOptedOut",
+        "help": "Whether to exclude users from analytics.",
+        "macrosInSelect": true,
+        "simpleValueType": true,
+        "type": "SELECT",
+        "selectItems": [
+          {
+            "displayValue": "False",
+            "value": false
+          },
+          {
+            "displayValue": "True",
+            "value": true
+          }
+        ]
+      },
+      {
+        "displayName": "Region",
+        "name": "region",
+        "help": "The DNS server to target (support values: de, us).",
+        "simpleValueType": true,
+        "type": "TEXT"
+      },
+      {
+        "displayName": "Cookie Duration",
+        "name": "cookieDuration",
+        "help": "The cookie duration in milliseconds (default to `15552000000`, 6 months).",
+        "simpleValueType": true,
+        "type": "TEXT",
+        "valueValidators": [
+          {
+            "type": "POSITIVE_NUMBER"
+          }
+        ]
+      },
+      {
+        "displayName": "User Token",
+        "name": "setUserToken",
+        "help": "Set a persistent user identifier applied to subsequent events.",
+        "simpleValueType": true,
+        "type": "TEXT"
```

Comment:
again, not sure how this can be used in this context, since it's a method, I would have expected it to appear on the same level as other methods `init`, `clickedObjectIDsAfterSearch`, etc

Reply:
That's a value at this point (I'll rename it to `userToken`). If set, it calls `setUserToken`.